tests/CLI/modules/file_tests.py

```diff
@@ -163,6 +163,49 @@ def test_volume_detail(self):
             ]
         }, json.loads(result.output))
 
+    def test_volume_detail_name_identifier(self):
+        result = self.run_command(['file', 'volume-detail', 'user'])
+
+        self.assert_no_fail(result)
+        self.assertEqual({
```

Comment:
We don't really need to test the WHOLE output is equal, that will make this test break anytime we change the fixture.

Instead you can do something like this to test a few fields of the output

```python
    self.assert_called_with('SoftLayer_Network_Storage', 'getObject')
    self.assertIn('username', result.output)
    self.assertIn('READHEAVY_TIER', result.output)
```
or whatever other strings you want to look for to make sure it called the correct data.